<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>music_player</class>
 <widget class="QMainWindow" name="music_player">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="music_player_back">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>390</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-image: url(:/images/music_back.svg);
background-repeat: no-repeat;
 background-position: center;
background-color: transparent;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="progress">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>741</width>
      <height>20</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"> QSlider::groove:horizontal {
           height: 4px; /* 设置滑动杆的高度 */
           background: #131A21; /* 设置滑动杆的背景颜色 */
           margin: 2px 0;
       }
       QSlider::handle:horizontal {
           width: 6px; /* 设置滑块的宽度 */
           background: #555; /* 设置滑块的背景颜色 */
           border: 1px solid #222; /* 设置滑块的边框 */
           margin: -5px 0; /* 设置滑块的外边距，使其居中 */
           border-radius: 3px; /* 设置滑块的圆角半径 */
       }
       QSlider::sub-page:horizontal {
           background: #E34C59; /* 设置已滑过区域的颜色 */
           border-radius: 5px; /* 设置已滑过区域的圆角半径 */
       }
       QSlider::add-page:horizontal {
          background: #2F2E31; /* 设置未滑过区域的颜色 */
          border-radius: 5px; /* 设置未滑过区域的圆角半径 */
       }</string>
    </property>
    <property name="maximum">
     <number>99</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_cd">
    <property name="geometry">
     <rect>
      <x>279</x>
      <y>70</y>
      <width>240</width>
      <height>240</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="volume">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>390</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-image: url(:/images/volume.svg);
background-repeat: no-repeat;
 background-position: center;
background-color: transparent;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="file_list">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-image: url(:/images/file_list.svg);
background-repeat: no-repeat;
 background-position: center;
background-color: transparent;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="mode">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>390</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-repeat: no-repeat;
 background-position: center;
background-color: transparent;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_song_name">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_progress">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>320</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_pointer">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>-90</y>
      <width>311</width>
      <height>240</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>380</y>
      <width>252</width>
      <height>72</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <widget class="QPushButton" name="previous">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-image: url(:/images/previous.svg);
background-repeat: no-repeat;
 background-position: center;
background-color: transparent;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="start_pause">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-repeat: no-repeat;
 background-position: center;
background-color: transparent;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="next">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-image: url(:/images/next.svg);
background-repeat: no-repeat;
 background-position: center;
background-color: transparent;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="volume_widget" native="true">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>269</y>
      <width>40</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #34343E;
border-radius: 10px;</string>
    </property>
    <widget class="QSlider" name="volume_Slider">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>10</y>
       <width>16</width>
       <height>91</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QSlider {
    background-color: transparent;
}

QSlider::groove:vertical {
    background: #EA3E3E;
    width: 8px;
    border-radius: 4px;
    border: none;
}

QSlider::handle:vertical {
    background: #FFFFFF;
    height: 20px;
    width: 20px;
    border-radius: 10px; /* 将滑块变成圆形 */
    margin: -5px -8px;
    border: none;
}

QSlider::sub-page:vertical {
    background: #494952;
    border: none;
}

QSlider::add-page:vertical {
    background: #EA3E3E;
    border: none;
}
</string>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="volume_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>97</y>
       <width>40</width>
       <height>17</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>10</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
